src/System.IO.Pipelines/ref/Configurations.props

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netstandard1.1;
+      netstandard;
+      netcoreapp;
```

Comment:
We only need the netstandard config here, so remove all of the rest since all platforms have the same surface area. No need to cross-compile the ref.